src/app/recipe/new/_components/AddButton.tsx

```diff
@@ -0,0 +1,14 @@
+import { Plus } from "@/components/icons";
+
+type Props = {
+  title: string;
+};
+
+export const AddButton: React.FC<Props> = ({ title }) => {
```

Comment:
他の部分含めてpropsの書き方全体で統一したいので定例で相談させてください。

```suggestion
export const AddButton: React.FC<Props> = ({ title ← この部分 }) => {
```